modules/vaos/spec/services/eps/draft_appointment_service_spec.rb

```diff
@@ -0,0 +1,183 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Eps::DraftAppointmentService do
+  subject { described_class.new(user) }
+
+  let(:user) { build(:user, :vaos) }
+  let(:appointments_service) { instance_double(VAOS::V2::AppointmentsService) }
+  let(:eps_provider_service) { instance_double(Eps::ProviderService) }
+  let(:eps_appointment_service) { instance_double(Eps::AppointmentService) }
+  let(:referral_id) { 'ref-123' }
+  let(:provider_id) { '9mN718pH' }
+  let(:pagination_params) { { page: 1, per_page: 10 } }
+  let(:referral_data) do
+    {
+      provider_id:,
+      appointment_type_id: 'ov',
+      start_date: '2025-01-01T00:00:00Z',
+      end_date: '2025-01-03T00:00:00Z'
+    }
+  end
+
+  before do
+    # Set up service mock instances
+    allow(VAOS::V2::AppointmentsService).to receive(:new).with(user).and_return(appointments_service)
```

Comment:
Why not use VCR cassettes? The other unit tests use them, so this test seems inconsistent. You can potentially use existing cassettes.